recipe/meta.yaml

```diff
@@ -29,10 +29,13 @@ requirements:
     - python
     - setuptools
     - numpy x.x
+    - jinja2
+    - cython
```

Comment:
I would remove these, they shouldn't actually be needed for the build to succeed